src/test/ui/methods/method-call-lifetime-args-unresolved.stderr

```diff
@@ -1,3 +1,18 @@
+warning: cannot specify lifetime arguments explicitly if late bound lifetime parameters are present
```

Comment:
So...this is actually more correct? We *are* trying to specially a late-bound lifetime in turbofish. It's just interesting this is now being emitted.